backend/lib/fetch/providers.py

```diff
@@ -84,7 +84,7 @@ def geocode_providers(providers, geocoder_name=GEOCODER, engine=connect.create_d
 
```

Comment:
Is this the correct function name and docstring now? Seems like it is expecting a list of addresses, not a list of provider dictionaries.

Reply:
Done

Reply:
👍